cmd/sgx_plugin/README.md

```diff
@@ -184,6 +186,13 @@ There is a kustomization for deploying everything:
 $ kubectl apply -k ${INTEL_DEVICE_PLUGINS_SRC}/deployments/sgx_plugin/overlays/epc-nfd/
 ```
 
+The second approach has a lesser deployment footprint. It does not require the certificate manager or NFD. Instead, it deploys a self-signed certificate for the SGX admission webhook, and a helper daemonset that creates a node label and advertises EPC capacity.
```

Comment:
but deploying the plugin only as described in this section without the operator has no dependency to `cert-manager`.  we could also add the comment about SGX admission webhook once we have it.

Reply:
How would you change the wording?

Reply:
how about: "The second approach has a lesser deployment footprint.  It does not deploy NFD but a helper daemonset that creates the 'SGX EPC capable' node label and advertises EPC capacity to the API server. "